### ISSUE TITLE
Build a baseline classifier using mixit/birdnet embeddings.

### ISSUE THREAD

**Issue by acmiyaguchi** [dsgt-birdclef/birdclef-2023#84]
Copy pasting from slack:

You can sync gs://birdclef-2023/data/processed/birdclef-2023/train_embeddings_consolidated_v1/ to your local directory and use spark to read process it.

```                                                                                
root
 |-- embedding: array (nullable = true)
 |    |-- element: double (containsNull = true)
 |-- predictions: array (nullable = true)
 |    |-- element: map (containsNull = true)
 |    |    |-- key: string
 |    |    |-- value: long (valueContainsNull = true)
 |-- track_name: string (nullable = true)
```

It has 6.6k distinct tracks, and it also has the tracks with mixit. Turns out i also duplicated the work of running birdnet stuff on the mp3 and the wav files (meh). This is missing about 10k of tracks, so I have to figure out what went wrong and also debug a few more things. This should be enough build a classifier on. It's a 10gb dataset, so it might be ~2gb in memory if you just keep a single channel. You might consider using spark for classification (https://spark.apache.org/docs/latest/ml-classification-regression.html#logistic-regression), or dumping out a derived dataset thats flatter so you can use it with another tool of your choice.
This also includes a bunch of non-birdcalls too, so you'll have to filter it so it only contains confident results. 

The pipeline should probably do something like the following in spark:

- filter down to the source track, or to the first channel of the mixit model
- explode predictions array
- apply sigmoid function to predictions, and keep the tracks that are birdcalls (50%?)
- parse the species from the track name
- save the dataframe with the embedding, the species, and the confidence

Then with this dataset, you should be able to feed this into sklearn, pyspark, pytorch, etc. Treat this as a multi-class prediction. Let me know if you have trouble starting on this, but this will end up being one of our baselines models probably.
